### ISSUE TITLE
The installer doesn't execute

### ISSUE THREAD

**Issue by Nik-9649** [BetterDiscord/Installer#36]
After downloading the new installer for windows I tried to run it but it doesn't work my cursor changes to the loading mode ig? but nothing work I tried changing the folder also tried disabling the anti-virus didn't work no luck with the compatibility mode either

**Comment by rauenzi** [comment]
What version of Windows are you on?

**Comment by Nik-9649** [comment]
> What version of Windows are you on?

![image](https://user-images.githubusercontent.com/61933761/113492451-d9149a00-94f0-11eb-9e5c-325244802e0f.png)

**Comment by TheRodrixHD** [comment]
I'm having the same issue can you help its not blocked by any antivirus program 
![image](https://user-images.githubusercontent.com/68759622/113500823-c79bb400-9529-11eb-8166-a9ca0c0db64e.png)

**Comment by MuppoMu** [comment]
Yeah its just not doing anything when I run it, I've uninstalled the old version before installing this one, still nothing. It runs and appears to work but makes no actual changes to discord. I've tried running w/ admin.

**Comment by Billandhispill** [comment]
I think I found a short work-around for this, I extracted it into a folder via 7Zip and launched the app from there and it worked. After it launched I didn't experience any other problems installing it.

**Comment by Bandi48** [comment]
I have the same problem. 
I have uninstalled the old Beta BD, all plugins and themes. I do not have a single antivirus running, even the windows defender and firewall is turned off. 
AND I also have Windows 10 Pro. Is it possible, that the new BD Discord Installer is not compatible with this windows build?

**Comment by Hajdew** [comment]
Same problem on 1909 Win10 Pro

**Comment by Roaring1** [comment]
So uh .. .How to fix ? Im still having this issue to this day . It's a struggle

**Comment by Nik-9649** [comment]
> So uh .. .How to fix ? Im still having this issue to this day . It's a struggle

my solution to that was manual install its pretty easy
